### DISCUSSION TITLE
👾 Programming language selection

### DISCUSSION THREAD

**Discussion by carlospepin23** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#1] [General]
Greetings! The purpose of this discussion is to give you the opportunity to share your thoughts on the qualities of both languages and provide your perspective on which one should be used. It is important to note that both languages are viable options for this project because they have Telnet compatible libraries.

Options:

- Python 8 🐍
- Rust 2 🦀

**Comment by keisanabria** [comment]
Python 8 🐍: Python is well-known for its simplicity and readability, making it an excellent choice for quick development and easy maintenance. Its extensive libraries and community support make it highly versatile, especially for projects that require rapid prototyping and scripting. Since it is already known amongst us, it would take less time to learn a new language and just focus on the goal of the project.
Rust 2 🦀: Rust is praised for its performance and memory safety, making it ideal for systems programming and applications where reliability and efficiency are crucial. Its strict compiler checks help prevent common programming errors, providing a robust option for developing secure and high-performance software. The problem is that it is much more different than Python 8 and not many of us know about it.
